exercises/static/exercises/follow_turtlebot/web-template/exercise.py

```diff
@@ -149,21 +149,21 @@ def generate_modules(self):
         # motors# hal_module.HAL.motors = imp.new_module("motors")
 
         # Add HAL functions
-        hal_module.HAL.get_frontal_image = self.hal.get_frontal_image
```

Comment:
undo all changes in this file when drone_wrapper backend is ready

Reply:
sure